restapi/src/main/java/io/stargate/web/docsapi/service/query/DocumentServiceUtils.java

```diff
@@ -78,7 +79,52 @@ public static long maxFieldDepth(String field) {
    * @param fields fields
    * @return max depth or empty if no fields
    */
-  public static OptionalLong maxFieldsDepth(Collection<String> fields) {
-    return fields.stream().mapToLong(DocumentServiceUtils::maxFieldDepth).max();
+  public static OptionalInt maxFieldsDepth(Collection<String> fields) {
+    return fields.stream().mapToInt(DocumentServiceUtils::maxFieldDepth).max();
+  }
+
+  /**
+   * Simple utility to return a string value from doc api {@link Row}.
+   *
+   * @param row Row
+   * @return String or null if a row column {@value QueryConstants#STRING_VALUE_COLUMN_NAME) is null.
+   */
+  public static String getStringFromRow(Row row) {
```

Comment:
nit: rename to `getString`? `Row` is obvious from the parameter type.... however, it's a matter of personal preference, I do not insist :)